src/processors/default/createNinaNftProcesor.ts

```diff
@@ -28,13 +29,17 @@ export const createNinaNfts: Processor = {
         platformId: MusicPlatformType.nina,
         tokenMetadataURI: metadataAccount.uri,
         tokenURI: metadataAccount.uri,
-        approved: true // all nina nfts approved by default
+        approved: true, // all nina nfts approved by default
+        chainId: ChainId.solana,
+        nftFactoryId: mintAddress
```

Comment:
will we need to backfill this nftFactoryId field on old nfts?

Reply:
we backfill all nftFactoryIds in the lens migration